doc/classes/Light2D.xml

```diff
@@ -340,46 +340,67 @@
 	</methods>
 	<members>
 		<member name="color" type="Color" setter="set_color" getter="get_color">
+			The Light2D's [Color].
 		</member>
 		<member name="editor_only" type="bool" setter="set_editor_only" getter="is_editor_only">
+			If [code]true[/code] Light2D will only appear when editing the scene. Default value: [code]false[/code].
 		</member>
 		<member name="enabled" type="bool" setter="set_enabled" getter="is_enabled">
+			If [code]true[/code] Light2D will emit light. Default value: [code]true[/code].
 		</member>
 		<member name="energy" type="float" setter="set_energy" getter="get_energy">
+			The Light2D's energy value. The larger the value, the stronger the light.
 		</member>
 		<member name="mode" type="int" setter="set_mode" getter="get_mode" enum="Light2D.Mode">
+			The Light2D's mode. See MODE_* constants for values.
 		</member>
 		<member name="offset" type="Vector2" setter="set_texture_offset" getter="get_texture_offset">
+			The offset of the Light2D's [code]texture[/code].
 		</member>
 		<member name="range_height" type="float" setter="set_height" getter="get_height">
+			The height of the Light2D. Used with 2D normal mapping.
 		</member>
 		<member name="range_item_cull_mask" type="int" setter="set_item_cull_mask" getter="get_item_cull_mask">
+			The layer mask. Only objects with a matching mask will be affected by the Light2D.
 		</member>
 		<member name="range_layer_max" type="int" setter="set_layer_range_max" getter="get_layer_range_max">
+			Maximum layer value of objects that are affected by the Light2D. Default value: [code]0[/code].
 		</member>
 		<member name="range_layer_min" type="int" setter="set_layer_range_min" getter="get_layer_range_min">
+			Minimum layer value of objects that are affected by the Light2D. Default value: [code]0[/code].
 		</member>
 		<member name="range_z_max" type="int" setter="set_z_range_max" getter="get_z_range_max">
+			Maximum [code]Z[/code] value of objects that are affected by the Light2D. Default value: [code]1024[/code].
```

Comment:
Lowercase z? I think most if not all properties in Godot are in lowercase.

Reply:
I debated that one. I was looking at the Inspector at the time and it's capital there.  I suppose lower is better because it's Node2D.z in code.

Reply:
Inspector can be set to capitalize property names.

Reply:
Wow, I had no idea that was a setting!